holidays/countries/comoros.py

```diff
@@ -0,0 +1,86 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from holidays.groups import ChristianHolidays, IslamicHolidays, InternationalHolidays
+from holidays.observed_holiday_base import ObservedHolidayBase
+
+
+class Comoros(ObservedHolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolidays):
+    """Comoros holidays.
+
+    References:
+        * [Public holidays in the Comoros](https://en.wikipedia.org/wiki/Public_holidays_in_the_Comoros)
+        * [UNION DES COMORES](https://web.archive.org/web/20240723035934/https://munganyo.km/public/storage/xnXxUFrh28LH1bQZpqAsLhGqXeearo-metaRGVjcmV0IE7CsDI0LTA1NlBSIGR1IDEwIGF2cmlsIDIwMjQgUG9ydGFudCBKb3VybsOpZSBjaG9tw6llIGV0IHBhecOpZS5wZGY=-.pdf)
```

Comment:
```suggestion
        * <https://en.wikipedia.org/wiki/Public_holidays_in_the_Comoros>
        * [Décret N°24-056PR](https://web.archive.org/web/20250717033820/https://munganyo.km/decrees/225)
        * [Décret N°25-008PR](https://web.archive.org/web/20250715223159/https://munganyo.km/decrees/374)
```
Special holidays Apr 13, 2024 and Jan 30, 2025.